data/test_endpoints/example_endpoints.yml

```diff
@@ -22,4 +22,3 @@ tracker_store:
   #db: rasa
   #user: username
   #password: password
-  
```

Comment:
@wochinge i'm not sure how to make this line reappear because on my local file it seems to be there but git isn't registering any changes :grumpycat: